syllabus/core/curriculum_base.py

```diff
@@ -5,28 +5,30 @@
 import numpy as np
 from gymnasium.spaces import Dict
 from syllabus.task_space import TaskSpace
+import random
 
 
 # TODO: Move non-generic logic to Uniform class. Allow subclasses to call super for generic error handling
 class Curriculum:
     """Base class and API for defining curricula to interface with Gym environments.
     """
 
-    def __init__(self, task_space: TaskSpace, random_start_tasks: int = 0, task_names: Callable = None) -> None:
+    def __init__(self, task_space: TaskSpace, task_names: Callable = None, use_startup_behavior: bool = False, sampled_strategy: str = None) -> None:
```

Comment:
```suggestion
    def __init__(self, task_space: TaskSpace, task_names: Callable = None, warmup_strategy: str = None, warmup_samples: int = 0) -> None:
```

I think I mentioned this before, but `use_startup_behavior` is unnecessary,, just have a strategy called "none" for sampled strategy.

Also I think renaming `sampled_strategy` to `warmup_strategy` is better, and allowing the user to configure the number of `warmup_samples` is a good idea.